test/WebJobs.Script.Tests.Integration/TestScripts/Node/HttpTriggerReturn/index.js

```diff
@@ -0,0 +1,3 @@
+﻿module.exports = function (context, req) {
+    context.res.type("text/plain").send("test");
```

Comment:
Yeah, I'd be onboard with changing the templates for node to not use $return. That way both programming models would work.

$return is most useful for C#/F# to avoid the complexities of out parameters. It particularly simplifies things for F#. However for Node, not a gain really.